Sources/Stagehand/AnimationQueue.swift

```diff
@@ -40,6 +41,19 @@ public final class AnimationQueue<ElementType: AnyObject> {
         }
     }
 
+    public var inProgressAnimationInstance: AnimationInstance? {
+        guard let currentAnimationInstance = queue.first?.instance else {
```

Comment:
This feels a little like we're exposing implementation details. How would you expect an adoptee to use this?